src/v3/src/components/AuthenticatorButton/AuthenticatorButton.tsx

```diff
@@ -207,49 +220,69 @@ const AuthenticatorButton: UISchemaElementComponent<{
             id={`${iconName}-usageDescription`}
             textAlign="start"
             sx={{
-              fontSize: '.875rem',
+              fontSize: tokens.TypographySizeSubordinate,
+              color: tokens.TypographyColorSubordinate,
               margin: 0,
```

Comment:
Following up on comment about using spacing tokens, we should ideally even update styling like this to use `tokens.Spacing0`

Reply:
Changed